src/Sentry.Log4Net/Sentry.Log4Net.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
to simplify dependency resolution, when targeting ns2.0 we should also target net461.

Also, that version of log4net doesn't support ns2.1 so there's no advantage to target ns2.1 here